### PR TITLE
Module page feedback

### PR DESCRIPTION
**Pull request by MarcinAman**

### CODE REVIEW

**Comment by MarcinAman** [comment]
For the reference, now it looks like this:
image

File: plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt [hunk 1]
```diff
@@ -308,22 +310,18 @@ open class DefaultPageCreator(
     protected open fun contentForComments(
         d: Documentable
     ): List<ContentNode> {
-        val tags: GroupedTags = d.documentation.flatMap { (pd, doc) ->
-            doc.children.asSequence().map { pd to it }.toList()
-        }.groupBy { it.second::class }
-
```
Comment (Kordyjan) [inline plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt:314]: I don't get this change. Now you are evaluating `groupedTags` every time you access it, so 7 times inside of this function. This is bad.

File: plugins/base/src/main/kotlin/translators/documentables/briefFromContentNodes.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
+package org.jetbrains.dokka.base.translators.documentables
+
+import org.jetbrains.dokka.pages.ContentGroup
+import org.jetbrains.dokka.pages.ContentNode
+import org.jetbrains.dokka.pages.ContentText
+import org.jetbrains.dokka.pages.TextStyle
+
+fun briefFromContentNodes(description: List<ContentNode>): List<ContentNode> {
+    val firstSentenceRegex = Regex("^((?:[^.?!]|[.!?](?!\\s))*[.!?])")
```
Comment (Kordyjan) [inline plugins/base/src/main/kotlin/translators/documentables/briefFromContentNodes.kt:9]:
```suggestion
    val firstSentenceRegex = """^((?:[^.?!]|[.!?](?!\s))*[.!?])""".toRegex()
```

Comment (Kordyjan) [reply plugins/base/src/main/kotlin/translators/documentables/briefFromContentNodes.kt:9]: I know it is only moved from another place, but this is not the idiomatic way of doing that.

**Review by Kordyjan — CHANGES_REQUESTED** [review]

File: plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt (hunk 1 above)
Comment (MarcinAman) [reply plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt:314]: Uh ok, my bad
changed to `val tags = d.groupedTags`

**Comment by sellmair** [comment]
Cherry picked into 1.4.0 🍒

**Pull request merged**